devices/utilities.c

```diff
@@ -110,18 +95,12 @@ int16_t *get_reg_ptr(Emulator *emu, uint8_t reg)
     Cpu *cpu = emu->cpu;
 
     static int16_t r2 = 0;
-    
+
     switch (reg)
-    {    
+    {
```

Comment:
if you would wish to submit this to the upstream, such changes (the amount of them) might be "frowned upon" by some... (granted - its their fault they're using IDE lacking basic "whitespace trimming", but still)

Reply:
As said before, upstreaming it does not seem feasible